### DISCUSSION TITLE
brew install stanc3

### DISCUSSION THREAD

**Discussion by ankane** [stan-dev/stanc3#1189] [Show and tell]
Hi, just wanted to share that stanc3 is now available on Homebrew, so Mac users can install it with:

```sh
brew install stanc3
```

Homebrew precompiles binaries (for both x86-64 and arm64), so installation is pretty much instant 🎉 

Ref: https://github.com/Homebrew/homebrew-core/pull/99930

**Comment by wds15** [comment]
Maybe put this on the forum?

Nice!

**Comment by rok-cesnovar** [comment]
Super cool @ankane!

**Comment by WardBrian** [comment]
This is very cool! Does it require someone to manually go and update it each version, or does it track github releases?

**Reply by ankane** [reply]
It's mostly manual right now (there's a bot that will sometimes submit version bumps), but between the Homebrew community and maintainers, Homebrew is pretty good about keeping things up-to-date. If you wanted to manually submit after releases, there's a `brew bump-formula-pr` command you can use.

**Comment by SteveBronder** [comment]
Awesome thank you so much!!
